src/components/reports/reportSummary/reportSummaryAlt.scss

```diff
@@ -0,0 +1,37 @@
+@import url("~@patternfly/patternfly/base/patternfly-variables.css");
+
+.chartSkeleton {
+  height: 175px;
+  margin-Bottom: var(--pf-global--spacer--md);
+  margin-Top: var(--pf-global--spacer--md);
+}
+
+.container {
+  display: flex;
+}
+
+.cost {
+  flex-Grow: 1;
+  min-Height: 470px;
+  margin-Right: var(--pf-global--spacer--md);
```

Comment:
I believe the properties should all be lower case (e.g.,  margin-right).

Reply:
fixed